google-cloud-spanner/src/main/java/com/google/cloud/spanner/encryption/CustomerManagedEncryption.java

```diff
@@ -17,29 +17,46 @@
 package com.google.cloud.spanner.encryption;
 
 import com.google.spanner.admin.database.v1.EncryptionConfig;
+import java.util.List;
 import java.util.Objects;
 
-/** The data is encrypted with a key provided by the customer. */
+/**
+ * The data is encrypted with a key provided by the customer.
+ */
 public class CustomerManagedEncryption implements BackupEncryptionConfig, RestoreEncryptionConfig {
 
   private final String kmsKeyName;
+  private final List<String> kmsKeyNames;
 
   CustomerManagedEncryption(String kmsKeyName) {
     this.kmsKeyName = kmsKeyName;
+    // TODO(harsha): Should this be [] by default?
+    this.kmsKeyNames = com.google.protobuf.LazyStringArrayList.EMPTY;
+  }
+
+  CustomerManagedEncryption(List<String> kmsKeyNames) {
+    this.kmsKeyNames = kmsKeyNames;
+    // TODO(harsha): Should this be "" by default?
+    this.kmsKeyName = "";
   }
 
```

Comment:
Maybe we can have CustomerManagedEncryption(String... kmsKeyNames), and check if only one keyName is being passed in, or multiple?